### PR TITLE
Ngfw 14560 : session kbps to bps changes

### PR DESCRIPTION
**Pull request by Shriram-Patil-Arista**
image

### CODE REVIEW

File: uvm/servlets/admin/app/model/Session.js [hunk 1]
```diff
@@ -28,14 +28,14 @@ Ext.define ('Ung.model.Session', {
         { name: 'remoteAddr', type: 'auto' },
         { name: 'serverCountry', type: 'string' },
         { name: 'serverIntf', type: 'auto' },
-        { name: 'serverKBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
+        { name: 'serverBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/app/model/Session.js:31]: Shouldn't this just be returning the value as-is now?

File: uvm/servlets/admin/app/model/Session.js [hunk 2]
```diff
@@ -28,14 +28,14 @@ Ext.define ('Ung.model.Session', {
...
         { name: 'serverLatitude', type: 'number' },
         { name: 'serverLongitude', type: 'number' },
         { name: 'sessionId', type: 'number' },
         { name: 'state', type: 'auto' },
         { name: 'tags', type: 'auto' },
         { name: 'tagsString', type: 'string' },
-        { name: 'totalKBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null ; } },
+        { name: 'totalBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null ; } },
```
Comment (cblaise-untangle) [inline uvm/servlets/admin/app/model/Session.js:38]: Shouldn't this just be returning the value as-is now?

File: uvm/servlets/admin/app/model/Session.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ Ext.define ('Ung.model.Session', {
         { name: 'bypassed', type: 'boolean' },
         { name: 'clientCountry', type: 'string' },
         { name: 'clientIntf', type: 'auto' },
-        { name: 'clientKBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
+        { name: 'clientBps', type: 'number', convert: function (val) { return val ? Math.round(val*1000)/1000 : null; } },
```
[the remark this answers is not in the document]
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/app/model/Session.js]: removed that part as it's not needed now

File: uvm/servlets/admin/app/model/Session.js (hunk 1 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/app/model/Session.js]: removed that part as it's not needed now

File: uvm/servlets/admin/app/model/Session.js (hunk 2 above)
Comment (Shriram-Patil-Arista) [reply uvm/servlets/admin/app/model/Session.js]: removed that part as it's not needed now

**Review by cblaise-untangle — APPROVED** [review]

**Pull request merged**